src/examples/live-data-basic/example-live-data-basic.component.ts

```diff
@@ -9,10 +9,11 @@ import { DataService, Quote } from '../data.service';
 	changeDetection: ChangeDetectionStrategy.OnPush,
 	encapsulation: ViewEncapsulation.None
 })
-export class ExampleLiveDataBasicComponent {
+export class ExampleLiveDataBasicComponent implements OnDestroy {
 	static id = 'live-data-basic';
 
 	rows: Quote[];
+	timerLink: any = null;
```

Comment:
default name is `timeoutId`
https://developer.mozilla.org/ru/docs/Web/API/WindowTimers/setTimeout